uhubctl.c

```diff
@@ -278,7 +281,31 @@ static int hub_port_status(struct libusb_device * dev, int nports, int portmask)
                 break;
             }
 
-            printf("   Port %d: %04x", port, port_status);
+            if (opt_verbose == 1) {
+                struct libusb_device *devi;
+                struct libusb_device_descriptor descr = { 0 };
```

Comment:
This causes compilation warning. Just avoid initialization completely:

    struct libusb_device_descriptor descr;